SPECS-EXTENDED/webkit2gtk3/webkit2gtk3.spec

```diff
@@ -0,0 +1,717 @@
+Vendor:         Microsoft Corporation
```

Comment:
Please lint the new spec files, so we have these tags in the right place.

Reply:
Some of the Requires need to be together due to associated comments.